Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:centos7
+LABEL maintainer="Nik Hodgkinson <11xor6@gmail.com>"
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.11.0
+RUN yum -y install epel-release && \
+    yum -y update && \
+    yum -y install java-1.11.0-openjdk-devel python-pip && \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    pip install --upgrade pip && \
+    pip install dumb-init
```

Comment:
do we need it? Why not to simply run `launcher`?

Reply:
Initially this was here to properly handle signals, however after further research into how `launcher` works, this seemingly isn't necessary.